kubernetes/samples/scripts/common/domain-template.yaml

```diff
@@ -63,7 +63,7 @@ spec:
     - name: JAVA_OPTIONS
       value: "%JAVA_OPTIONS%"
     - name: USER_MEM_ARGS
-      value: "-Djava.security.egd=file:/dev/./urandom "
+      value: "-Djava.security.egd=file:/dev/./urandom -Xms256m -Xmx512m "
```

Comment:
What about resource requests/limits?

Reply:
Oh, I see below... we already had those flags. You can ignore this.

Reply:
My understanding is that this template is used by domain-home-in-image and domain-on-pv samples and resource requests/limits section in domain resource is generated based on how below parameters are configured in create-domain-inputs.yaml file.

serverPodMemoryRequest
serverPodCpuRequest
serverPodMemoryLimit
serverPodCpuLimit